modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -21,10 +18,10 @@ class AppointmentsController < VAOS::BaseController
       def index
         appointments
 
-        _include&.include?('clinics') && merge_clinics(appointments[:data])
-        _include&.include?('facilities') && merge_facilities(appointments[:data])
-
         appointments[:data].each do |appt|
+          if include_params[:facilities] && appt[:location_id].present? && appt[:location].nil?
```

Comment:
I moved this logic out of the service and into the controller because the mobile app can't use the same error message. It would break the schema. I looked through the code carefully, and you can safely assume that nil indicates an error. If it doesn't error, it returns a struct. A struct with nil in it is still not nil. 

I should have done the same the same thing for provider names, but instead I added the default string and removed it for mobile. I'll try to correct that later.

Reply:
That all makes sense, no concerns on my end with this change in light of that info